guides-and-standards/standard-public-repos.md

```diff
@@ -0,0 +1,45 @@
+# Standards for public GitHub repositories at Arcadia
+
+This document describes the standards for public GitHub repositories at Arcadia. These standards are intended to ensure that the code we release is well-documented, reproducible, and easy to use. All Arcadia GitHub repositories should meet these standards prior to being made public.
+
+## Aside about repository templates
+
+We have Arcadia-specific templates for [Python](https://github.com/Arcadia-Science/python-analysis-template), [Snakemake](https://github.com/Arcadia-Science/snakemake-template), and [R](https://github.com/Arcadia-Science/r-analysis-template) repositories. It is strongly recommended to start new projects using these templates, as this will mean that your repository will automatically satisfy many of the requirements below.
+
+## Standards for organization and documentation
+
+- **Repository organization**: The repository should be well-organized with descriptive file names and a clear directory structure.
+  - Scripts, notebooks, and data files should generally be in separate directories.
+  - Directory names and filenames should not include spaces or special characters.
+  - Avoid very long scripts or notebooks. If a script or notebook is more than 500-1000 lines long, consider breaking it up into smaller scripts or functions.
```

Comment:
consider providing an annotated typical project structure, something like:

```
ArcadiaPublicProject/
├── assets/                 # Non-data static files (usually images or other media)
├── data/                   # Data files
│   ├── raw/                # Original, unmodified data
│   └── processed/          # Data that has been modified
├── docs/                   # Documentation files
├── lib/                    # External dependencies not brought in with standard package managers
├── notebooks/              # Jupyter notebooks for experiments or tutorials
│   ├── algorithms.ipynb
│   └── demo.ipynb
├── scripts/                # Standalone scripts for setup, installation, data import, etc.
│   └── setup_db.sh
├── src/                    # Source code files for the project
│   ├── main.py
│   ├── utils.py
│   └── test_utils.py
├── .gitignore              # Project environment files
├── LICENSE
└── README.md

```

Reply:
I like this but I feel it could be confusing since it's partially redundant with the template repos, so for now I just linked to those. Ideally in the future we'll have a proper template for python packages which would use and document this organization.